app/src/main/java/com/example/classroomScheduler/repository/HubListDao.kt

```diff
@@ -0,0 +1,60 @@
+package com.example.classroomScheduler.repository
+
+import android.content.ContentValues.TAG
+import android.util.Log
+import androidx.lifecycle.MutableLiveData
+import com.example.classroomScheduler.model.HubModel
+import com.google.firebase.auth.ktx.auth
+import com.google.firebase.firestore.FirebaseFirestore
+import com.google.firebase.ktx.Firebase
+import kotlinx.coroutines.Dispatchers
+import kotlinx.coroutines.GlobalScope
+import kotlinx.coroutines.launch
+
+class HubListDao {
+    val db = FirebaseFirestore.getInstance()
```

Comment:
Actually, I could, but then it would not follow the MVVM architecture. Otherwise too, this class is the only class responsible which is accessing the Firestore database to fetch the data related to HubList. Creating another class just for getting a Firestore instance would increase the code complexity.